plugins/modules/database_mysql_v2.py

```diff
@@ -326,12 +326,46 @@ def _handle_absent(self) -> None:
 
             self.register_action(f"Deleted MySQL database {database.id}")
 
+    def _handle_suspend(self) -> None:
+        params = self.module.params
+
+        database = safe_find(
+            self.client.database.mysql_instances,
+            MySQLDatabase.label == params.get("label"),
+        )
+
+        if database is not None:
+            self._populate_results(database)
+
+            database.suspend()
+
+            self.register_action(f"Suspended MySQL database {database.id}")
+
+    def _handle_resume(self) -> None:
+        params = self.module.params
+
+        database = safe_find(
+            self.client.database.mysql_instances,
+            MySQLDatabase.label == params.get("label"),
+        )
+
+        if database is not None:
+            self._populate_results(database)
+
+            database.resume()
+
+            self.register_action(f"Resumed MySQL database {database.id}")
```

Comment:
Optional thing for consideration, is it worth to combine some parts of these functions?

```suggestion
    def _handle_suspend_or_resume(self, state: str) -> None:
        params = self.module.params

        database = safe_find(
            self.client.database.mysql_instances,
            MySQLDatabase.label == params.get("label"),
        )

        if database is not None:
            self._populate_results(database)
            
            if state == "suspend":
                _handle_suspend(self, database)
            elif state == "resume":
                _handle_resume(self, database)

    def _handle_suspend(self, database) -> None:
        database.suspend()
        self.register_action(f"Suspended MySQL database {database.id}")

    def _handle_resume(self, database) -> None:
        database.resume()
        self.register_action(f"Resumed MySQL database {database.id}")
```